application-tour-ui/src/main/resources/TourCode/TourJson.xml

```diff
@@ -110,13 +119,19 @@
           #if ($element == 'body')
             #set ($element = '')
           #end
+          #if ($element == '')
+            #set ($orphan = true)
+          #else
+            #set ($orphan = false)
+          #end
           #set ($step = { 'element'  : $element,
                           'title'    : $services.localization.render($tourDoc.getValue('title', $stepObj)),
                           'content'  : $content,
                           'placement': $tourDoc.getValue('placement', $stepObj),
                           'order'    : $tourDoc.getValue('order', $stepObj),
                           'backdrop' : $backdrop,
-                          'path'     : $path })
+                          'path'     : $path, 
+                          'orphan'   : $orphan})
```

Comment:
Yeah, it's an override. I suppose you can ommit the `orphan: false` in the global config as it will have no effect.